### ISSUE TITLE
Commands do not work in Android 10

### ISSUE THREAD

**Issue by yellowbluesky** [yellowbluesky/PixivforMuzei3#91]
Android 10 added limitations on applications starting activities in the background.

Muzei in their 4.0 API release added a workaround for this, providing RemoteActionCompat for my application to consume

https://github.com/romannurik/muzei/pull/652

**Comment by yellowbluesky** [comment]
Work will proceed in a branch, will merge into master only when Muzei 4.0 API is released

**Comment by yellowbluesky** [comment]
Commands that work with new Muzei 3.4 Android 10 RemoteActionCompat commands:

- Sharing artwork

Commands that do not work with the new RemoteActionCompat:

- Bookmarking an image.
This command requires a network call, so I am using an IntentService for its background thread execution. The IntentService takes a while to fire (I verified that it was firing with a Notification), and then the network call doesn't appear to fire. The network call itself is the exact same code as the working "Add to bookmark" command.
I tried logging within onHandleIntent(), but no logs were being printed. Work is ongoing
